webapp/src/main/webapp/src/assets/i18n/en.json

```diff
@@ -1,7 +1,7 @@
 {
   "welcome": "Welcome {{firstName}} {{lastName}}",
   "application": {
-    "title": "Student Group Administration"
+    "title": "Manage Student Groups"
```

Comment:
This is the real change, everything else is unrelated while I'm already in this project.